Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -21,9 +21,10 @@ constexpr int EFB_SCALE_AUTO_INTEGRAL = 0;
 
 enum class AspectMode : int
 {
-  Auto,
-  AnalogWide,
-  Analog,
+  Auto,           // 4:3 or 16:9
+  ForceWide,      // 16:9
+  ForceStandard,  // 4:3
+  Custom,
   Stretch,
```

Comment:
Please don't reorder enum values that are saved to settings. This will cause everyone who previously had Stretch selected to now have Custom selected.

Reply:
I was aware of that, I didn't think it mattered much as it's probably just a few people that would actually be the ones that'd be interested in this feature.
I can move `Custom` after stretch though.

Reply:
You'll need to swap the labels in the UI now too, they're wrong in the current version of the PR.